Error when pushing

This issue happens when I try to apply `git push origin HEAD`:
```
> devtools-core@0.0.1 prepush /Users/seanlee/Projects/react_learning/devtools-core
> yarn run lint && yarn run flow

yarn run v1.3.2
warning package.json: No license field
$ yarn run lint-css -s; a=$?; yarn run lint-js -s; b=$?; test $a -eq 0 -a $b -eq 0
warning package.json: No license field
$ stylelint packages/devtools-launchpad/**/*.css -s
Error: You must use a valid syntax option, either: scss, less or sugarss
    at getCode.then.code (/Users/seanlee/Projects/react_learning/devtools-core/node_modules/stylelint/lib/getPostcssResult.js:57:15)
    at <anonymous>
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
warning package.json: No license field
$ eslint packages/devtools-launchpad/src; eslint packages/devtools-reps/src; eslint packages/devtools-components/src -s
Invalid option '-s' - perhaps you meant '-h'?
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! devtools-core@0.0.1 prepush: `yarn run lint && yarn run flow`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the devtools-core@0.0.1 prepush script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/seanlee/.npm/_logs/2017-12-15T22_47_51_944Z-debug.log

husky - pre-push hook failed (add --no-verify to bypass)

error: failed to push some refs to 'git@github.com:weilonge/devtools-core.git'

```

--- a/package.json
+++ b/package.json
@@ -5,11 +5,11 @@
   "scripts": {
     "flow": "flow",
     "license-check": "lerna run license-check",
-    "lint": "yarn run lint-css; a=$?; yarn run lint-js; b=$?; test $a -eq 0 -a $b -eq 0",
+    "lint": "yarn lint-css; a=$?; yarn lint-js; b=$?; test $a -eq 0 -a $b -eq 0",
     "lint-css": "stylelint packages/devtools-launchpad/**/*.css",
     "lint-js": "eslint packages/devtools-launchpad/src; eslint packages/devtools-reps/src; eslint packages/devtools-components/src",
     "nom": "rm -rf node_modules/ packages/*/node_modules/ && yarn install",
-    "prepush": "yarn run lint && yarn run flow",
+    "prepush": "yarn lint && yarn flow",
     "postinstall": "lerna bootstrap",
     "test": "lerna run test"
   },

--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
   "scripts": {
     "flow": "flow",
     "license-check": "lerna run license-check",
-    "lint": "yarn run lint-css -s; a=$?; yarn run lint-js -s; b=$?; test $a -eq 0 -a $b -eq 0",
+    "lint": "yarn run lint-css; a=$?; yarn run lint-js; b=$?; test $a -eq 0 -a $b -eq 0",
     "lint-css": "stylelint packages/devtools-launchpad/**/*.css",
     "lint-js": "eslint packages/devtools-launchpad/src; eslint packages/devtools-reps/src; eslint packages/devtools-components/src",
     "nom": "rm -rf node_modules/ packages/*/node_modules/ && yarn install",

